src/lib/utils/exception.hpp

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include <iostream>
+#include <exception>
+#include <stdexcept>
+#include <sstream>
+
+namespace opossum {
+
+class InvalidInput : public std::runtime_error {
```

Comment:
I appreciate that InvalidInput is small, but please use the same name for the file anyway. Think of the people who look for classes using their file browser.

Reply:
also pls doc that this is used by AssertInput

Reply:
Thought this could be a file like assert.hpp, where we have all exceptions in one place (since they are usually very short). Or do you think we don't need this?

Reply:
Don't get me wrong, but things that *could* be something if *someone in the future* does it, rarely turn out to actually *be* that particular something :)

We can think about a unified exception compile unit if there are actually multiple hyrise-defined exceptions, until then its just a normal class with a normal file name